chemprop/cli/train.py

```diff
@@ -50,6 +50,9 @@
 logger = logging.getLogger(__name__)
 
 
+_CV_DEP_WARNING = "The -k/--num-folds argument is deprecated and will be completely removed in v2.2.0 - use --num-replicates instead."
```

Comment:
A depreciation warning sounds like --num-folds should still work, but you set this message to be displayed with an argument error. I think it okay to give an error with this message when the flag is given in v.2.1 and then remove the error message and flag in v2.2, but is there a stronger word than "deprecated", something like:
```suggestion
_CV_DEP_WARNING = "The -k/--num-folds argument was removed in v2.1.0, please use --num-replicates instead."
```

Reply:
Good point - did this in all places where relevant in c852b75